go.mod

```diff
@@ -185,6 +186,7 @@ require (
 	github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d // indirect
 	github.com/apache/thrift v0.16.0 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bits-and-blooms/bitset v1.14.3
```

Comment:
Why do we need another one after we already have `FastIntSet`?

Reply:
That one is for the sparsed case. When the elements are dense, it's not very suitable.
The bitset has an operation like `SetAll` which the sparse set will not have.

Reply:
Or we just say that
- if we want to use a more high-performance struct to replace `[]bool`, we need the bitset.
- if we want to use a more high-performance struct to replace `map[int]struct{}`, we need the `FastIntSet`

Reply:
`FastIntSet` provides optimization for sizes less than 64, which is just essentially a bitset.
And it also provides `AddRange()` method, which is even easier than `SetAll`.

Reply:
No, they are two different things.
![image](https://github.com/user-attachments/assets/69d0c2de-44d0-499c-85c4-f1b9465f8b80)
The `FastIntSet` is actually a set. Not a bitmask compression for `[]bool`.